content/platforms/kubernetes/managing-images.md

```diff
@@ -0,0 +1,295 @@
+your K8s cluster, you may be able to pull the images without credentials. For
+example, a GKE cluster in the same project as the GCR registry will have the
+necessary authorization to pull the images. Otherwise, you may
+need to add a [pull secret](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) to your namespace
+and then tell Kubernetes and the operator to use the pull secret.
+
+If you need to specify credentials, each of the following sections will
+detail where the pull secret needs to be specified.
+
+### Specifying the operator image
+
+The operator bundle (e.g., see the [6.0.8-1 bundle.yaml](https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/6.0.8-1/bundle.yaml))
+contains the operator deployment and the reference to the operator image (redislabs/operator). To use a private repository, you must
+change this image reference before you deploy the operator. This image
+should point to the same value tag pushed to the private repository:
+
+```
+${PRIVATE_REPO}/redislabs/operator:${OPERATOR_VERSION}
+```
+
+For example:
+
+```YAML
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis-enterprise-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: redis-enterprise-operator
+  template:
+    metadata:
+      labels:
+        name: redis-enterprise-operator
+    spec:
+      serviceAccountName: redis-enterprise-operator
+      containers:
+        - name: redis-enterprise-operator
+          image: gcr.io/yourproject/redislabs/operator:6.0.8-1
+...
+```
+
+Note that if you apply this change to modify an existing operator deployment,
+the operator's pod will restart. As the operator is stateless, it will pull
+the image from the private repository (as necessary) and restart the pod. This
+kind of change will not affect any existing cluster managed by the operator.
+
+If your registry requires a pull secret, the standard `imagePullSecrets`
+may be specified on the operator deployment:
+
+
+```YAML
+spec:
+  template:
+    spec:
+      imagePullSecrets:
+      - name: regcred
+```
+
+### Specifying the Redis Enterprise cluster images
+
+A Redis Enterprise cluster managed by the operator consists of three
+container images:
+
+ * **`redislabs/redis`** - the Redis Enterprise container image
+ * **`redislabs/operator`** - the bootstrapper container image packaged with the operator
+ * **`redislabs/k8s-controller`** - the service rigger container image
+
+By default, when a new cluster is created by the operator, it uses the
+standard container images listed above. As such, it will not necessarily pull
+them from the same private repository from which the operator may have been
+pulled. The images will be pulled from the default container registry for
+the Kubernetes cluster (i.e., typically DockerHub).
+
+The consequence is that you must specify all three container images
+to pull all the images from the same private repository. The container
```

Comment:
Can you rephrase the first sentence and be more specific?

Reply:
I reworked this whole paragraph and the preceding paragraph.